scripts/bootstrap-nixos.sh

```diff
@@ -277,7 +280,7 @@ if yes_or_no "Run nixos-anywhere installation?"; then
 	nixos_anywhere
 fi
 
-if [ -z "${luks_secondary_drive_labels}"]; then
```

Comment:
As far as I can tell, this was inverted from the intended behavior? `[ -z "$luks_secondary_drive_labels" ]` meant `setup_luks_secondary_drive_decryption` would only be run if --luks-secondary-drive-labels didn't have anything passed to it.